packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -6086,6 +6063,15 @@ function escapeStringForLinkHeaderQuotedParamValueContextReplacer(
   }
 }
 
+export function hoistHoistables(
+  target: HoistableState,
+  source: HoistableState,
+) {
+  target.charset.push(...source.charset);
```

Comment:
Let's not use spread since it'll get transpiled to a form that we don't know and likely worse than the alternative we'd do manually.